src/protocols/rdp/keymaps/dk_dk_qwerty.keymap

```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+parent  "base"
+name    "dk-dk-qwerty"
```

Comment:
The format used for the name is similar to that used for locales: the two-letter language code, two-letter country code, and finally keyboard variant. The final "-dk-qwerty" looks correct, but perhaps this should be "da-dk-qwerty"? "dk" is the country code for Denmark, but not the language code for Danish.